wp-content/themes/mojintranet/page_media_grid.php

```diff
@@ -1,49 +1,60 @@
-<?php if (!defined('ABSPATH')) die();
-
+<?php if (!defined('ABSPATH')) {
```

Comment:
You would usually have this on a single line, as it's just a way to prevent direct access.